### PR TITLE
Translated and corrected to Russian curriculum/challenges/russian/01-responsive-web-design/applied-accessibility/

### PR DESCRIPTION
**Pull request by kontramot**
- [x ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [х] My pull request targets the `master` branch of freeCodeCamp.
- [х] None of my changes are plagiarized from another source without proper attribution.
- [х] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Review by greggubarev — CHANGES_REQUESTED** [review]
Please imrpove translate at line

- [ ] 10

**Review by greggubarev — APPROVED** [review]
### LGTM 👍  Congrats on your first contribution! 
Image of Office
Thank you for the PR ✨ 🎉 We're happy to accept these changes and look forward to your future contributions.

Happy coding!

File: curriculum/challenges/russian/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.russian.md [hunk 1]
```diff
@@ -11,10 +11,15 @@ localeTitle: Добавьте текстовое описание для изо
...
 
+
 ## Instructions (Задание)
 <section id="instructions"> Camper Cat, настоящий ниндзя в жизни и в программировании, создает веб-сайт, чтобы поделиться своими знаниями. Он хочет использовать изображение для профиля, которое продемонстрирует его навыки и будет оценено всеми посетителями сайта. Добавьте атрибут <code>alt</code> в тег <code>img</code> так, чтобы всем стало понятно, что Camper Cat занимается каратэ. (Изображение <code>src</code> не ссылается на фактический файл, поэтому вы должны увидеть текст <code>alt</code> на дисплее.) </section>
+## Description
```
Comment (ValeraS) [inline curriculum/challenges/russian/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.russian.md]: @greggubarev You have made 2 sections of the description. Please, fix this.

File: curriculum/challenges/russian/01-responsive-web-design/applied-accessibility/avoid-colorblindness-issues-by-carefully-choosing-colors-that-convey-information.russian.md [hunk 2]
```diff
@@ -8,10 +8,15 @@ localeTitle: 'Будьте внимательны к пользователям,
...
 
-# Instructions (Задание)
-<section id="instructions"> Camper Cat пробует различные стили для важной кнопки, но желтый ( <code>#FFFF33</code> ) <code>background-color</code> и зеленый ( <code>#33FF33</code> ) текст <code>color</code> находятся рядом на цветовом круге и практически неразличимы для некоторых пользователей-дальтоников. (По этой же причине сложно проверить их коэффициент контрастности - он слишком низкий). Измените цвет <code>color</code> текста на синий ( <code>#003366</code> ), чтобы решить обе проблемы одновременно. </section>
 
-## Tests (Тесты)
+## Description
```
Comment (ValeraS) [inline curriculum/challenges/russian/01-responsive-web-design/applied-accessibility/avoid-colorblindness-issues-by-carefully-choosing-colors-that-convey-information.russian.md]: @greggubarev You have made 2 sections of the description. Please, fix this.

File: curriculum/challenges/russian/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.russian.md (hunk 1 above)
Comment (greggubarev) [reply curriculum/challenges/russian/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.russian.md]: @ValeraS I`m need to create a new PR - for fix it?

Comment (ValeraS) [reply curriculum/challenges/russian/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.russian.md]: Yes

**Pull request merged**